mtc/README.md

```diff
@@ -0,0 +1,139 @@
+# OpenShift Cluster Application Migration Tool Lab
+
+This guide will provide you with hands-on exposure to the new OpenShift Migration Tooling through a series of labs migrating application workloads (both stateful and stateless) between OpenShift Clusters.
+
+## Description
+
+Migrates application workloads from Cluster A to Cluster B (includes ui, controller, Velero/Restic, plugins, etc). This application is available for installation through the Operator Hub/OLM. This course requires both the CAM OCP3 and OCP4 services from RHPDS.
+
+## Goals
+
+* Migrate several application workloads (both stateful and stateless) from an OpenShift 3.11 cluster to an OpenShift 4.1 cluster using the Cluster Application Migration Tool (CAM)
```

Comment:
"from an OpenShift 3.11 cluster to an OpenShift 4.1"

to

"from an OpenShift 3.11 cluster to an OpenShift 4.x"

Reply:
Fixed in daa0a48